src/fragments/ui/auth/flutter/authenticator.mdx

```diff
@@ -0,0 +1,335 @@
+## Basic Integration
+
+The Amplify-Flutter Authenticator Widget is a Material-compatible sign-up and sign-in experience which reacts to your Amplify configuration. It automatically detects the authentication state and updates the UI. If the user is signed in, the underlying component (typically your app's home widget) is displayed; otherwise, signin/signup controls are displayed.
+
+To begin, follow the steps for configuring the Amplify Auth category from the [getting started walkthrough](/lib/auth/getting-started). Be sure to also add the Amplify-Flutter Authenticator to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  amplify_authenticator: ^1.0.0-rc.0
+```
+
+You will also need to import the Amplify-Flutter Authenticator into your app:
+
+```dart
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+```
+
+Then, update your main app widget's `build` method:
+
+```dart
+@override
+	Widget build(BuildContext context) {
+	  return MaterialApp(
+	    title: 'Authenticator Demo',
+	    // The Authenticator component should be a descendant of your MaterialApp.
+	    home: Authenticator(
+	      child: const Scaffold(
+	        body: Center(
+	          child: Text('You are logged in!'),
+	        ),
+	      ),
+	    ),
+	  );
+	}
+```
+
+If you run your application, you should now see the Amplify-Flutter Authenticator Widget!
+
+
+![Basic SignIn](/images/flutterAuthenticator/flutter_authenticator_signin_base.png)
+
+## Customizable Forms
+
+Typically, the Amplify-Flutter Authenticator will use your Amplify configuration to determine which inputs are displayed in a form. It also offers the ability to override this default behavior and customize three of its UI forms:
+
+- SignInForm
+- SignUpForm
+- ConfirmSignInNewPasswordForm
```

Comment:
Maybe link to API docs

Reply:
Do you mean for the top level Authenticator API, or the individual form classes?

Reply:
I'm thinking the actual form APIs so they can see the constructor and the different fields and stuff

Reply:
Good suggestion - I did the same for the text resolvers, although we seem to have not gotten auto generated docs for message resolver.